### ISSUE TITLE
anomal-area

### ISSUE THREAD

**Issue by lzzlxxlsz** [LeeDoYup/AnoGAN#23]
errors = np.mean(np.squeeze(errors),axis=2)
the error is the anomal-score image,and this function of the code is that using the gray-level to label the anomal-area?

**Comment by lzzlxxlsz** [comment]
You said that the anomaly_score is the sum of the all pixels anomaly_score,if I want to see the every pixel's anomaly_score,what shoul I do ? 
the code"self.res_loss = tf.reduce_mean(
        tf.reduce_sum(tf.abs(tf.subtract(test_inputs, self.ano_G))))" is  the whole pixel's res_loss ?it is 64*64 and 3 channels,but for the dis_loss : "self.dis_loss = tf.reduce_mean(
          tf.reduce_sum(tf.abs(tf.subtract(dis_f_z, dis_f_input))))",it is 6*6 and 512 channels?,what should I do if I want to know every pixel?
